modules/openxr/openxr_interface.cpp

```diff
@@ -90,7 +98,9 @@ PackedStringArray OpenXRInterface::get_suggested_tracker_names() const {
 		"/user/vive_tracker_htcx/role/waist",
 		"/user/vive_tracker_htcx/role/chest",
 		"/user/vive_tracker_htcx/role/camera",
-		"/user/vive_tracker_htcx/role/keyboard"
+		"/user/vive_tracker_htcx/role/keyboard",
+
```

Comment:
Is the newline intentional?

Reply:
Yeah just trying to create blocks between our normal paths, our vive tracker paths and the paths for the new interaction.